crates/bevy_render/src/texture/image_texture_conversion.rs

```diff
@@ -1,152 +1,158 @@
+            }
+            image::DynamicImage::ImageBgra8(i) => {
+                width = i.width();
+                height = i.height();
+                format = TextureFormat::Bgra8UnormSrgb;
 
-            let raw_data = i.into_raw();
+                data = i.into_raw();
+            }
+            image::DynamicImage::ImageLuma16(i) => {
+                width = i.width();
+                height = i.height();
+                format = TextureFormat::R16Uint;
 
-            data = raw_data.as_slice().as_bytes().to_owned();
-        }
-        image::DynamicImage::ImageLumaA16(i) => {
-            width = i.width();
-            height = i.height();
-            format = TextureFormat::Rg16Uint;
+                let raw_data = i.into_raw();
 
-            let raw_data = i.into_raw();
+                data = raw_data.as_slice().as_bytes().to_owned();
+            }
+            image::DynamicImage::ImageLumaA16(i) => {
+                width = i.width();
+                height = i.height();
+                format = TextureFormat::Rg16Uint;
 
-            data = raw_data.as_slice().as_bytes().to_owned();
-        }
+                let raw_data = i.into_raw();
 
-        image::DynamicImage::ImageRgb16(image) => {
-            width = image.width();
-            height = image.height();
-            format = TextureFormat::Rgba16Uint;
-
-            let mut local_data =
-                Vec::with_capacity(width as usize * height as usize * format.pixel_size());
-
-            for pixel in image.into_raw().chunks_exact(3) {
-                // TODO unsafe_get in release builds?
-                let r = pixel[0];
-                let g = pixel[1];
-                let b = pixel[2];
-                let a = u16::max_value();
-
-                local_data.extend_from_slice(&r.to_ne_bytes());
-                local_data.extend_from_slice(&g.to_ne_bytes());
-                local_data.extend_from_slice(&b.to_ne_bytes());
-                local_data.extend_from_slice(&a.to_ne_bytes());
+                data = raw_data.as_slice().as_bytes().to_owned();
             }
 
-            data = local_data;
-        }
-        image::DynamicImage::ImageRgba16(i) => {
-            width = i.width();
-            height = i.height();
-            format = TextureFormat::Rgba16Uint;
+            image::DynamicImage::ImageRgb16(image) => {
+                width = image.width();
+                height = image.height();
+                format = TextureFormat::Rgba16Uint;
+
+                let mut local_data =
+                    Vec::with_capacity(width as usize * height as usize * format.pixel_size());
 
-            let raw_data = i.into_raw();
+                for pixel in image.into_raw().chunks_exact(3) {
+                    // TODO unsafe_get in release builds?
+                    let r = pixel[0];
+                    let g = pixel[1];
+                    let b = pixel[2];
+                    let a = u16::max_value();
```

Comment:
```suggestion
                    let a = u16::MAX;
```
`max_value()` is deprecated in favor of `::MAX`